packages/block-editor/src/store/selectors.js

```diff
@@ -187,7 +187,7 @@ export const __unstableGetBlockWithoutInnerBlocks = createSelector(
 export function getBlocks( state, rootClientId ) {
 	const treeKey =
 		! rootClientId || ! areInnerBlocksControlled( state, rootClientId )
-			? rootClientId || ''
+			? rootClientId || ROOT_CONTAINER_CLIENT_ID
```

Comment:
While the introduction of this constant can have value, it's adding a lot of noise to the PR, ideally these things are done in dedicated PRs IMO.

Reply:
Yeah, that's the plan. I was having a hard time understanding all the code with the random empty strings. Once things are working, we can extract that part into its own PR.